packages/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -181,6 +184,15 @@ export class InternalDropdownItem extends React.Component<InternalDropdownItemPr
       ) : (
         childNode
       );
+    const preventedEvents = inoperableEvents.reduce(
```

Comment:
Nice! Since it's essentially a same capability as we use in [Button](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/Button/Button.tsx#L101), and may be needed on other components as well, can we maybe abstract this out into a utility function that can be imported/shared between?

Reply:
Yup, sure can! 🙂